config/local_docker/.paths.env

```diff
@@ -1,7 +1,8 @@
+BIN_DIR=/usr/local/bin
 CONTRACTS_DIR=/specular/contracts
 CONFIG_DIR=/specular/config
-
-# Define binaries
+OPS_GENESIS_BIN=/usr/local/bin/genesis
```

Comment:
should this be in other config folders?

Reply:
It is an inheritance issue that needs to be fixed. I can check again.